docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -0,0 +1,325 @@
+
+
+#### Supported and unsupported updates
+
+ - You can add new MBeans or resources simply by specifying their corresponding model file YAML snippet along with their parent bean hierarchy. See [Example of adding a data source](#example-of-adding-a-datasource).
+
+ - You can recreate, change, or add secrets that your model depends on. For example, you can change a database password secret.
+
+ - You can change or add environment variables that your model macros may depend on (macros that use the `@@ENV:myenvvar@@` syntax).
+
+ - You can remove a named MBean or resource by specifying a model file with a `!` symbol just before the bean or resource name. For example, if you have a data source named `mynewdatasource` defined in your model, it can be removed by specifying a small model file that loads after the model file that defines the data source, where the small model file looks like this:
+
+  ```
+  resources:
+    JDBCSystemResource:
+      !mynewdatasource:
+  ```
+
+  For more information, see [Declaring Named MBeans to Delete](https://github.com/oracle/weblogic-deploy-tooling#declaring-named-mbeans-to-delete) in the WebLogic Deploying Tooling documentation.
+
+ - You can add or alter an MBean attribute by specifying a YAML snippet along with its parent bean hierarchy that references an existing MBean and the attribute. For example, to add or alter the max capacity of a data source named `mynewdatasource`:
+
+  ```
+  resources:
+    JDBCSystemResource:
+      mynewdatasource:
+        JdbcResource:
+          JDBCConnectionPoolParams:
+            MaxCapacity: 5
+  ```
+
+  For more information, see [Using Multiple Models](https://github.com/oracle/weblogic-deploy-tooling#using-multiple-models) in the WebLogic Deploy Tooling documentation.
+
+ - There is no way to directly delete an attribute from an MBean that's already been specified by a model file. The work-around is to do this using two model files: (a) add a model file that deletes the named bean/resource that is a parent to the attribute you want to delete, and (b) add another  subsequent model file that fully defines the named bean/resource but without the attribute you want to delete.
+
+ - The following runtime updates haven't been tested and are _not_ supported in the first release of Model in Image. If you need to make these kinds of updates, consider shutting down your domain entirely before making the change:
+   - Adding, removing, or altering the network configuration of an existing Managed Server. This includes, but isn't limited to network channels, ports, and cluster addresses.
+   - Adding a Managed Server to an existing configured cluster.
```

Comment:
Any reason it won't work?  I believe QA has tested adding cluster.  The thing I don't see here is some of these changes may require changing the domain resource yaml.   The primordial domain only has admin server and each update including the first deployment, then the python will do the scan and pass back control to operator.  So, if a new cluster is added, then the domain resource need to have a new element for it.

Reply:
I'm okay with saying that something is not supported to limit our testing requirement :)

Reply:
As far as I know, none of this is getting tested.  That's a V2 plan.

Reply:
Closing.  Started internal slack discussion at thread https://proddev-paas-fmw.slack.com/archives/CKFA73SNM/p1585093723144200